src/reducers/profileReducer.js

```diff
@@ -15,9 +17,20 @@ const initialState = {
         barChart:{
 
          }
-    }
+    },
+    paginatedProfiles:[],
+    startIndex:0
 };
 
+const getAllButRejectedProfiles = profiles => {
+    return profiles.filter(profile => profile.status != "Rejected");
+}
+
+const applyPagination = (activePage, profiles) => {
+    return getAllButRejectedProfiles(profiles).slice((activePage-1) * constants.pageSize,
+       (constants.pageSize * activePage))
+}
+
 const activeProfile = ["Application Received", "Interview Scheduled", "Offer Made", "On Hold"]
```

Comment:
I think we repeat this array many times so it is the right time to move this is into constant

Reply:
This array has been moved to the constants file